pkg/apis/kops/util/versions_test.go

```diff
@@ -21,21 +21,54 @@ import (
 )
 
 func Test_ParseKubernetesVersion(t *testing.T) {
-	grid := map[string]string{
-		"1.3.7":         "1.3.7",
-		"v1.4.0-beta.8": "1.4.0-beta.8",
-		"1.5.0":         "1.5.0",
-		"https://storage.googleapis.com/kubernetes-release-dev/ci/v1.4.0-alpha.2.677+ea69570f61af8e/": "1.4.0",
+	cases := []struct {
+		version       string
+		expected      string
+		expectedError bool
+	}{
+		{
+			version:       "1.3.7",
+			expected:      "1.3.7",
+			expectedError: false,
+		},
+		{
+			version:       "v1.4.0-beta.8",
+			expected:      "1.4.0-beta.8",
+			expectedError: false,
+		},
+		{
+			version:       "1.5.0",
+			expected:      "1.5.0",
+			expectedError: false,
+		},
+		{
+			version:       "https://storage.googleapis.com/kubernetes-release-dev/ci/v1.4.0-alpha.2.677+ea69570f61af8e/",
+			expected:      "1.4.0",
+			expectedError: false,
+		},
+		{
+			version:       "",
+			expected:      "",
```

Comment:
You don't need to explicitly list `expected: ""` for cases that expect an error.

Reply:
done